core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -332,6 +343,8 @@ object TopicCommand extends Logging {
         allTopicLevelOpts -- Set(describeOpt) + reportUnderReplicatedPartitionsOpt + topicsWithOverridesOpt)
       CommandLineUtils.checkInvalidArgs(parser, options, topicsWithOverridesOpt,
         allTopicLevelOpts -- Set(describeOpt) + reportUnderReplicatedPartitionsOpt + reportUnavailablePartitionsOpt)
+      CommandLineUtils.checkInvalidArgs(parser, options, ifExistsOpt, allTopicLevelOpts -- Set(alterOpt, deleteOpt))
```

Comment:
I get why `listOpt` wouldn't be included here, but why not `describeOpt`? Seems to me like it would be in the same list as `alterOpt` and `deleteOpt` since all use `getTopics`. Is this simply because we weren't throwing `IllegalArgumentException`s before? I guess we're stuck with that for compatibility, although it could be nice if you got an error if you specified a filter pattern that resulted in 0 topics.